### ISSUE TITLE
ViewPager

### ISSUE THREAD

**Issue by 4nd3250n** [Prototik/HoloEverywhere#193]
Is there any chance to see a ViewPager ported to Holoeverywhere?

**Comment by Prototik** [comment]
Are you have any problems to use ViewPager?

**Comment by httpdispatch** [comment]
I'm using android.support.v4.view.ViewPager with HoloEverywhere without a problems

**Comment by 4nd3250n** [comment]
Hmmm,

At first I thought it might be so.
I had an activity, inside a view pager which held a ListView. ListView items were inflated from a xml layout file. The problem was that TextView inside an item did not have correct color. It turned out I was pass application context instead of activity context. The question still remains how to correctly change theme of an application at runtime? Right now I am doing it in each activity separately in onCreate() event. 

Bottom line: apart from the question about theme setting this issue can be deleted.
